e2e/crd_validation_test.go

```diff
@@ -240,6 +240,57 @@ func TestCRDDefaulting(t *testing.T) {
 					},
 				},
 			},
+			"TargetLabels/from-kubernetes-labels": {
+				obj: &monitoringv1.ClusterPodMonitoring{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "nondefault",
+						Namespace: "default",
+					},
+					Spec: monitoringv1.ClusterPodMonitoringSpec{
+						Endpoints: []monitoringv1.ScrapeEndpoint{
+							{
+								Interval: "1m",
+								Port:     intstr.FromString("metrics"),
+							},
+						},
+						TargetLabels: monitoringv1.ClusterTargetLabels{
+							FromPod: []monitoringv1.LabelMapping{
+								{
+									From: "app.kubernetes.io/name",
+									To:   "k8s_app_name",
+								},
+								{
+									From: "app.kubernetes.io/instance",
```

Comment:
suggestion: have a `fromPod` relabeling that omits `to` entirely to test the implicit to-labeling.